pkg/descheduler/strategies/lownodeutilization.go

```diff
@@ -269,6 +281,21 @@ func evictPods(inputPods []*v1.Pod,
 	}
 }
 
+// podToleratesTaints returns true if a pod tolerates one node's taints
+func podToleratesTaints(pod *v1.Pod, taintsOfNodes map[string][]v1.Taint) bool {
+	canBeEvicted := true
```

Comment:
Just a nit though this way it's shorter and easier to read given it's sufficient to find first node where the pod tolerates the taints:
```
	for nodeName, taintsForNode := range taintsOfNodes {
		if len(pod.Spec.Tolerations) >= len(taintsForNode) && helperUtil.TolerationsTolerateTaintsWithFilter(pod.Spec.Tolerations, taintsForNode, nil) {
				return true
		}
		klog.V(5).Infof("pod: %#v doesn't tolerate node %s's taints", pod.Name, nodeName)
	}

	return false
```